geps/gep-713.md

```diff
@@ -649,6 +1135,29 @@ const (
 )
 ```
 
+#### On targeted resources
+
+Implementations that use Policy objects MUST ensure that a Condition exists in
+the `status.Conditions` of any objects affected by a Policy. If that Condition is
+already set to `status: true`, then the implementation must ensure that
+the `observedGeneration` field matches the `metadata.generation` field of the
+object.
```

Comment:
I think this specific solution might be best represented by a follow-up GEP. There are a lot of edge cases here that are going to be hard to handle. For example it will be fairly straightforward for implementations to determine when they should _set_ this condition, but near impossible for them to determine when they should _unset_ the condition, assuming more than one implementation is active in the cluster.

Reply:
I've updated the wording here and in the Solution Cookbook section, but I'll also update both with the "this needs a followup GEP" wording and change to Experimental.